SetupDataPkg/SetupDataDxe/SchemaXmlHashDriver/SchemaXmlHash.c

```diff
@@ -0,0 +1,80 @@
+/** @file SchemaXmlHash.c
+  Set config xml hash to NVRAM variable.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Base.h>
+#include <ConfigStdStructDefs.h>
+#include <Generated/ConfigClientGenerated.h>
+#include <Generated/ConfigDataGenerated.h>
+#include <Generated/ConfigProfilesGenerated.h>

```

Comment:
Instead of directly pulling in the autogen headers, you can `#include <Library/PlatformConfigDataLib.h>`.

Note that the library header needs to be updated to include the hash you variable you newly injected.

Please also update the null library instance to reflect the change as well.

Reply:
i removed this file in the latest commit. thanks